src/controllers/monitorController.js

```diff
@@ -14,9 +14,10 @@ const getMonitors = async (req, res) => {
 };
 
 const addMonitor = async (req, res) => {
-  const { endpoint_key, schedule } = req.body;
+  const { schedule } = req.body;
```

Comment:
@marymcdonald to answer your slack question, for the time being the route just expects a schedule. And thank you for explaining the ID, I placed it back in and removed it as something the frontend needs to pass